staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -640,3 +688,22 @@ func (in crdStorageMap) clone() crdStorageMap {
 	}
 	return out
 }
+
+type CRDRESTOptionsGetterWrapper struct {
```

Comment:
`NoConversionRESTOptionsGetter` would be a better name. Wdyt?

Reply:
It is not necessarily NoConversion, `CrdConversionRESTOptionGetter` is better I guess.

Reply:
Ok, sounds fine